config/config.go

```diff
@@ -1005,10 +1005,11 @@ type KubernetesRole string
 
 // The valid options for KubernetesRole.
 const (
-	KubernetesRoleNode     = "node"
-	KubernetesRolePod      = "pod"
-	KubernetesRoleService  = "service"
-	KubernetesRoleEndpoint = "endpoints"
+	KubernetesRoleNode     = KubernetesRole("node")
+	KubernetesRolePod      = KubernetesRole("pod")
+	KubernetesRoleService  = KubernetesRole("service")
+	KubernetesRoleEndpoint = KubernetesRole("endpoints")
+	KubernetesRoleIngress  = KubernetesRole("ingress")
```

Comment:
No need to change, but `KubernetesRoleIngress KubernetesRole = "ingress"` is a bit more common and would save a character ;-)